### PR TITLE
upgrade amplify packages + fixes

### PR DESCRIPTION
**Pull request by Job-madi**

### CODE REVIEW

File: amplify/backend/function/probatereviewawsclient/parameters.json
Comment (cdrake) [inline amplify/backend/function/probatereviewawsclient/parameters.json:1]: Why does this say nodejs14.x?

File: src/app/unreviewed-detail/unreviewed-detail.component.ts [hunk 1]
```diff
@@ -1590,13 +1591,17 @@ export class UnreviewedDetailComponent implements OnInit {
...
   editLineItemByIndex(index: number): void {
+    console.log('Starting line item by index');
     let lineItem: LineItem | null = null;
     if (this.record && this.record.lineItems) {
+      console.log('in clause')
       lineItem = this.record.lineItems.items[index];
     }
 
     if (!lineItem) {
+      console.log('throwing error***************THROWING ERROR!!!!!!!!!!!!!');
```
Comment (cdrake) [inline src/app/unreviewed-detail/unreviewed-detail.component.ts:1602]: this should be removed

**Review by cdrake — APPROVED** [review]
LGTM. Just saw one console.log that probably be removed.  Up to you.

File: amplify/backend/function/probatereviewawsclient/parameters.json
Comment (Job-madi) [reply amplify/backend/function/probatereviewawsclient/parameters.json:1]: Seems like that's the node runtime version the functions are using. I'll look at updating it in a separate PR.

**Pull request merged**